src/main/resources/resources/languages/languages_en_US.properties

```diff
@@ -87,3 +90,9 @@ panel.save=Save
 panel.open=Open
 panel.saveModifiedQuestionMessage=The current file has been modified.\nDo you want to save the changes before closing?
 panel.unencryptedDataWarningMessage=Please note that all data will be stored unencrypted.\nMake sure you keep the exported file in a secure location.
+
+language.languageChanged=Language has changed successfully.
+language.en.us=English
```

Comment:
I don't think it is a good idea to store language names in language resources. Every time a new translation in introduced, then all the resources need to be updated. Maybe it would be better to hard-code them in the supported languages map.